django/cantusdb_project/main_app/views/chant.py

```diff
@@ -805,13 +805,15 @@ def get_initial(self):
             }
         latest_folio = latest_chant.folio if latest_chant.folio else "001r"
```

Comment:
I'm wondering if it would make more sense to used `date_created` as the way to find the "latest chant" rather than `date_updated`. 

Let's say I create a chant, then realize a chant I created 5 folios back needs an edit, make that edit, and then come back to creating chants. Then, `latest_chant` won't be what I want. But if you went by `date_created` then you would get the "right" chant.

Maybe there is a case where `date_updated` leads to the "correct" result more often? 

Of course, in the case I mentioned one can just make the edits manually, but this seem like a simple change that might make more sense?

Reply:
Looking back at the original issue, I'm now interpreting the request as neither of these options: pre-populating the `Office` field with the previous chant in the source (in other words, determined by folio and sequence rather than by when updated or created).

I